tests/components/rainmachine/test_config_flow.py

```diff
@@ -57,6 +51,39 @@ async def test_invalid_password(hass):
         assert result["errors"] == {CONF_PASSWORD: "invalid_auth"}
 
 
+async def test_options_flow(hass):
+    """Test config flow options."""
+    conf = {
+        CONF_IP_ADDRESS: "192.168.1.100",
+        CONF_PASSWORD: "password",
+        CONF_PORT: 8080,
+        CONF_SSL: True,
+    }
+
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        unique_id="abcde12345",
+        data=conf,
+        options={CONF_ZONE_RUN_TIME: 900},
+    )
+    config_entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.rainmachine.async_setup_entry", return_value=True
+    ):
+        result = await hass.config_entries.options.async_init(config_entry.entry_id)
+
+        assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+        assert result["step_id"] == "init"
+
+        result = await hass.config_entries.options.async_configure(
+            result["flow_id"], user_input={CONF_ZONE_RUN_TIME: 600}
+        )
+
+        assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
+        assert config_entry.options == {CONF_ZONE_RUN_TIME: 600}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)